text2graph/schema.py

```diff
@@ -89,30 +92,29 @@ class Location(BaseModel):
     lat: Annotated[float, AfterValidator(valid_latitude)] | None = None
     lon: Annotated[float, AfterValidator(valid_longitude)] | None = None
 
-    def model_post_init(self, __context) -> None:
-        self.hydrate()
-
-    def hydrate(self) -> None:
+    async def hydrate(self) -> None:
         "Hydrate Location from SERPAPI"
         serpapi_key = os.environ["SERPAPI_KEY"]
```

Comment:
Admittedly, I should have done this originally, but I think it makes more sense to abstract the guts of this method to a function that returns (lat, lon) in text2graph.geolocation.serpapi